diff --git a/Cargo.toml b/Cargo.toml
index 0efcc2e58..432c7d51b 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -216,7 +216,7 @@ nu-protocol = { path = "./crates/nu-protocol", version = "0.105.1" }
 nu-std = { path = "./crates/nu-std", version = "0.105.1" }
 nu-system = { path = "./crates/nu-system", version = "0.105.1" }
 nu-utils = { path = "./crates/nu-utils", version = "0.105.1" }
-reedline = { workspace = true, features = ["bashisms", "sqlite"] }
+reedline = { workspace = true, features = ["bashisms", "sqlite-dynlib"] }

 crossterm = { workspace = true }
 ctrlc = { workspace = true }
@@ -279,7 +279,7 @@ default = [
   "plugin",
   "trash-support",
   "sqlite",
-  "rustls-tls"
+  "native-tls"
 ]
 stable = ["default"]
 # NOTE: individual features are also passed to `nu-cmd-lang` that uses them to generate the feature matrix in the `version` command
diff --git a/crates/nu-cli/Cargo.toml b/crates/nu-cli/Cargo.toml
index 1b0911c73..3ceed10fe 100644
--- a/crates/nu-cli/Cargo.toml
+++ b/crates/nu-cli/Cargo.toml
@@ -29,7 +29,7 @@ nu-protocol = { path = "../nu-protocol", version = "0.105.1", features = ["os"]
 nu-utils = { path = "../nu-utils", version = "0.105.1" }
 nu-color-config = { path = "../nu-color-config", version = "0.105.1" }
 nu-ansi-term = { workspace = true }
-reedline = { workspace = true, features = ["bashisms", "sqlite"] }
+reedline = { workspace = true, features = ["bashisms", "sqlite-dynlib"] }

 chrono = { default-features = false, features = ["std"], workspace = true }
 crossterm = { workspace = true }
diff --git a/crates/nu-command/Cargo.toml b/crates/nu-command/Cargo.toml
index 5082c1d38..52bf6a74e 100644
--- a/crates/nu-command/Cargo.toml
+++ b/crates/nu-command/Cargo.toml
@@ -168,7 +168,7 @@ features = [
 workspace = true

 [features]
-default = ["os", "rustls-tls"]
+default = ["os", "native-tls"]
 os = [
 	# include other features
 	"js",
