From b639d4f487ce87ea008d048a25343db107f6bffe Mon Sep 17 00:00:00 2001
From: socksinspace <socks-in.space@mailbox.org>
Date: Fri, 23 May 2025 16:05:09 +0200
Subject: [PATCH] bootstrap fix

---
 ltmain.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ltmain.sh b/ltmain.sh
index 1a710177..2747ed24 100644
--- a/ltmain.sh
+++ b/ltmain.sh
@@ -6028,7 +6028,7 @@ func_mode_link ()
 	      if test -n "$inst_prefix_dir"; then
 		case $libdir in
 		  [\\/]*)
-		    add_dir="$add_dir -L$inst_prefix_dir$libdir"
+		    add_dir=" -L$inst_prefix_dir$libdir"
 		    ;;
 		esac
 	      fi
-- 
2.49.0

