--- a/packaging/standalone/standalone-community/src/main/distribution/text/community/conf/neo4j.conf
+++ b/packaging/standalone/standalone-community/src/main/distribution/text/community/conf/neo4j.conf
@@ -9,11 +9,11 @@
 #initial.dbms.default_database=neo4j
 
 # Paths of directories in the installation.
-#server.directories.data=data
-#server.directories.plugins=plugins
-#server.directories.logs=logs
-#server.directories.lib=lib
-#server.directories.run=run
+server.directories.data=/var/lib/neo4j/data
+server.directories.plugins=/var/lib/neo4j/plugins
+server.directories.logs=/var/log/neo4j
+server.directories.lib=/usr/share/neo4j/lib
+server.directories.run=/run/neo4j
 #server.directories.licenses=licenses
 #server.directories.transaction.logs.root=data/transactions
 
@@ -155,27 +155,27 @@ server.https.enabled=false
 
 # Bolt SSL configuration
 #dbms.ssl.policy.bolt.enabled=true
-#dbms.ssl.policy.bolt.base_directory=certificates/bolt
+dbms.ssl.policy.bolt.base_directory=/etc/neo4j/certs/bolt
 #dbms.ssl.policy.bolt.private_key=private.key
 #dbms.ssl.policy.bolt.public_certificate=public.crt
 #dbms.ssl.policy.bolt.client_auth=NONE
 
 # Https SSL configuration
 #dbms.ssl.policy.https.enabled=true
-#dbms.ssl.policy.https.base_directory=certificates/https
+dbms.ssl.policy.https.base_directory=/etc/neo4j/certs/https
 #dbms.ssl.policy.https.private_key=private.key
 #dbms.ssl.policy.https.public_certificate=public.crt
 #dbms.ssl.policy.https.client_auth=NONE
 
 # Cluster SSL configuration
 #dbms.ssl.policy.cluster.enabled=true
-#dbms.ssl.policy.cluster.base_directory=certificates/cluster
+dbms.ssl.policy.cluster.base_directory=/etc/neo4j/certs/cluster
 #dbms.ssl.policy.cluster.private_key=private.key
 #dbms.ssl.policy.cluster.public_certificate=public.crt
 
 # Backup SSL configuration
 #dbms.ssl.policy.backup.enabled=true
-#dbms.ssl.policy.backup.base_directory=certificates/backup
+dbms.ssl.policy.backup.base_directory=/etc/neo4j/certs/backup
 #dbms.ssl.policy.backup.private_key=private.key
 #dbms.ssl.policy.backup.public_certificate=public.crt
 
@@ -334,17 +334,6 @@ server.jvm.additional=-Dlog4j.layout.jsonTemplate.maxStringLength=32768
 # Useful for big queries and big expressions where cypher code generation can create large methods.
 #server.jvm.additional=-XX:-DontCompileHugeMethods
 
-#********************************************************************
-# Wrapper Windows NT/2000/XP Service Properties
-#********************************************************************
-# WARNING - Do not modify any of these properties when an application
-#  using this configuration file has been installed as a service.
-#  Please uninstall the service before modifying this section.  The
-#  service can then be reinstalled.
-
-# Name of the service
-server.windows_service_name=neo4j
-
 #********************************************************************
 # Other Neo4j system properties
 #********************************************************************
