--- a/cmake/Pcsx2Utils.cmake
+++ b/cmake/Pcsx2Utils.cmake
@@ -44,9 +44,9 @@
 endfunction()
 
 function(get_git_version_info)
-	set(PCSX2_GIT_REV "")
-	set(PCSX2_GIT_TAG "")
-	set(PCSX2_GIT_HASH "")
+	set(PCSX2_GIT_REV "" PARENT_SCOPE)
+	set(PCSX2_GIT_TAG "" PARENT_SCOPE)
+	set(PCSX2_GIT_HASH "" PARENT_SCOPE)
 	if (GIT_FOUND AND EXISTS ${PROJECT_SOURCE_DIR}/.git)
 		EXECUTE_PROCESS(WORKING_DIRECTORY ${PROJECT_SOURCE_DIR} COMMAND ${GIT_EXECUTABLE} describe --tags
 			OUTPUT_VARIABLE PCSX2_GIT_REV
@@ -87,7 +87,7 @@
 			OUTPUT_STRIP_TRAILING_WHITESPACE
 			ERROR_QUIET)
 		if (NOT PCSX2_GIT_REV)
-			set(PCSX2_GIT_REV "Unknown")
+			set(PCSX2_GIT_REV "v2.3.26")
 		endif()
 	endif()
 
