* Unmerged path pyproject.toml
diff --git a/setup.py b/setup.py
index 549f09d..d37be23 100644
--- a/setup.py
+++ b/setup.py
@@ -8,7 +8,6 @@ import tarfile
 import sysconfig
 import subprocess
 
-from cmake import CMAKE_BIN_DIR
 from setuptools import setup
 from setuptools import find_packages
 from setuptools import Extension
@@ -47,13 +46,6 @@ if PLATFORM == 'darwin':
     # if archflags is given we build in specific subdir
     if ARCHFLAGS:
         BUILD_DIR = 'build_' + '_'.join(ARCHFLAGS)
-YASM_VERSION = '1.3.0'
-YASM_SOURCE = 'yasm-%s.tar.gz' % YASM_VERSION
-YASM_URL = 'https://github.com/yasm/yasm/releases/download/v%s/' % YASM_VERSION + YASM_SOURCE
-JPEG_VERSION = '3.0.3'
-JPEG_SOURCE = 'libjpeg-turbo-%s.tar.gz' % JPEG_VERSION
-JPEG_URL = 'https://github.com/libjpeg-turbo/libjpeg-turbo/archive/%s.tar.gz' % JPEG_VERSION
-
 
 def untar_url(url, filename):
     path = filename.rstrip('.tar.gz')
@@ -68,12 +60,6 @@ def untar_url(url, filename):
             t.extractall(pt.dirname(filename))
     return path
 
-
-# download sources
-YASM_DIR = untar_url(YASM_URL, pt.join(PACKAGE_DIR, 'lib', YASM_SOURCE))
-JPEG_DIR = untar_url(JPEG_URL, pt.join(PACKAGE_DIR, 'lib', JPEG_SOURCE))
-
-
 def cvar(name):
     return sysconfig.get_config_var(name)
 
@@ -130,27 +116,6 @@ class cmake_build_ext(build_ext):
             sys.stderr = sys.stdout
             # MSVC build environment
             update_env_msvc()
-        self.build_cmake_dependency(YASM_DIR, [
-            '-DBUILD_SHARED_LIBS=OFF'
-        ])
-        cflags = ''
-        if PLATFORM == 'linux':
-            # make GCC put functions and data in separate sections
-            # the linker can then remove unused sections to reduce the size of the binary
-            cflags = '-ffunction-sections -fdata-sections '
-        env = {
-            # add YASM to the path
-            'PATH': pt.join(YASM_DIR, BUILD_DIR) + os.pathsep + os.getenv('PATH', ''),
-            # custom CFLAGS - depends on platform
-            'CFLAGS': cflags + os.getenv('CFLAGS', ''),
-        }
-        self.build_cmake_dependency(JPEG_DIR, [
-            *flags,
-            '-DWITH_CRT_DLL=1',  # fixes https://bugs.python.org/issue24872
-            '-DENABLE_SHARED=0',
-            '-DREQUIRE_SIMD=1',
-            '-DCMAKE_POSITION_INDEPENDENT_CODE=ON',
-        ], env=env)
         # build extensions
         super().run()
 
@@ -163,7 +128,7 @@ class cmake_build_ext(build_ext):
         os.chdir(build_dir)
         config = 'Debug' if self.debug else 'Release'
         env = dict(os.environ, **(env or {}))
-        cmake = os.path.join(CMAKE_BIN_DIR, 'cmake')
+        cmake = 'cmake'
         subprocess.check_call([
             cmake,
             '-G' + make_type(), '-Wno-dev',
@@ -180,25 +145,13 @@ class cmake_build_ext(build_ext):
 
 
 def _libdir():
-    return pt.join(JPEG_DIR, BUILD_DIR)
-
-
-def _staticlib():
-    if PLATFORM in ('linux', 'darwin'):
-        return 'libturbojpeg.a'
-    elif PLATFORM == 'windows':
-        return 'turbojpeg-static.lib'
-    else:
-        raise RuntimeError('Platform not supported: %s, %s' % (PLATFORM, ARCH))
-
+    return pt.join(BUILD_DIR)
 
 def make_jpeg_module():
     include_dirs = [
         NumpyImport(),
-        pt.join(JPEG_DIR),
         pt.join(PACKAGE_DIR, 'simplejpeg'),
     ]
-    static_libs = [pt.join(_libdir(), _staticlib())]
     cython_files = [pt.join('simplejpeg', '_jpeg.pyx')]
     for cython_file in cython_files:
         if pt.exists(cython_file):
@@ -226,8 +179,8 @@ def make_jpeg_module():
         'simplejpeg._jpeg',
         sources,
         language='C',
+        libraries=['turbojpeg'],
         include_dirs=include_dirs,
-        extra_objects=static_libs,
         extra_link_args=extra_link_args,
         extra_compile_args=extra_compile_args,
         define_macros=macros,
@@ -312,8 +265,6 @@ class ConcatFiles:
 
 LICENSE_FILES = [
     'LICENSE',
-    pt.join(JPEG_DIR, 'LICENSE.md'),
-    pt.join(JPEG_DIR, 'README.ijg')
 ]
 with ConcatFiles(*LICENSE_FILES):
     setup(
* Unmerged path simplejpeg/_jpeg.c
