diff --git a/configure.ac b/configure.ac
index af4c95f..458ca08 100644
--- a/configure.ac
+++ b/configure.ac
@@ -30,7 +30,7 @@ AC_ARG_ENABLE(pda_folders,

 dnl ---- Check for libmb so we can figure out PNG / XPM support ------------

-PKG_CHECK_MODULES(LIBMB, libmb >= 1.1,, AC_MSG_ERROR([*** libmb required and not found ***]))
+# PKG_CHECK_MODULES(LIBMB, libmb >= 1.1,, AC_MSG_ERROR([*** libmb required and not found ***]))

 # below is quite bad ...
 if $PKG_CONFIG --libs libmb | grep png ; then
