--- a/psycopg/psycopg/_compat.py
+++ b/psycopg/psycopg/_compat.py
@@ -33,17 +33,17 @@ else:
 if sys.version_info >= (3, 10):
     from typing import TypeGuard, TypeAlias
 else:
-    from typing_extensions import TypeGuard, TypeAlias
+    from typing import TypeGuard, TypeAlias
 
 if sys.version_info >= (3, 11):
     from typing import LiteralString, Self
 else:
-    from typing_extensions import LiteralString, Self
+    from typing import LiteralString, Self
 
 if sys.version_info >= (3, 13):
     from typing import TypeVar
 else:
-    from typing_extensions import TypeVar
+    from typing import TypeVar
 
 __all__ = [
     "Counter",
--- a/tests/utils.py
+++ b/tests/utils.py
@@ -21,9 +21,9 @@ if sys.version_info >= (3, 11):
 
     assert_type = typing.assert_type
 else:
-    import typing_extensions
+    import typing
 
-    assert_type = typing_extensions.assert_type
+    assert_type = typing.assert_type
 
 
 import pytest
--- a/psycopg_pool/psycopg_pool/_compat.py
+++ b/psycopg_pool/psycopg_pool/_compat.py
@@ -18,17 +18,17 @@ else:
 if sys.version_info >= (3, 10):
     from typing import TypeAlias
 else:
-    from typing_extensions import TypeAlias
+    from typing import TypeAlias
 
 if sys.version_info >= (3, 11):
     from typing import Self
 else:
-    from typing_extensions import Self
+    from typing import Self
 
 if sys.version_info >= (3, 13):
     from typing import TypeVar
 else:
-    from typing_extensions import TypeVar
+    from typing import TypeVar
 
 __all__ = [
     "Counter",
