diff --git a/crates/supermaven_api/src/supermaven_api.rs b/crates/supermaven_api/src/supermaven_api.rs
index 3dede69..3dcbe18 100644
--- a/crates/supermaven_api/src/supermaven_api.rs
+++ b/crates/supermaven_api/src/supermaven_api.rs
@@ -226,66 +226,7 @@ pub async fn has_version(version_path: &Path) -> bool {
 }
 
 pub async fn get_supermaven_agent_path(client: Arc<dyn HttpClient>) -> Result<PathBuf> {
-    fs::create_dir_all(supermaven_dir())
-        .await
-        .with_context(|| {
-            format!(
-                "Could not create Supermaven Agent Directory at {:?}",
-                supermaven_dir()
-            )
-        })?;
-
-    let platform = match std::env::consts::OS {
-        "macos" => "darwin",
-        "windows" => "windows",
-        "linux" => "linux",
-        _ => return Err(anyhow!("unsupported platform")),
-    };
-
-    let arch = match std::env::consts::ARCH {
-        "x86_64" => "amd64",
-        "aarch64" => "arm64",
-        _ => return Err(anyhow!("unsupported architecture")),
-    };
-
-    let download_info = latest_release(client.clone(), platform, arch).await?;
-
-    let binary_path = version_path(download_info.version);
-
-    if has_version(&binary_path).await {
-        return Ok(binary_path);
-    }
-
-    let request = HttpRequest::get(&download_info.download_url);
-
-    let mut response = client
-        .send(request.body(AsyncBody::default())?)
-        .await
-        .with_context(|| "Unable to download Supermaven Agent".to_string())?;
-
-    let mut file = File::create(&binary_path)
-        .await
-        .with_context(|| format!("Unable to create file at {:?}", binary_path))?;
-
-    futures::io::copy(BufReader::new(response.body_mut()), &mut file)
-        .await
-        .with_context(|| format!("Unable to write binary to file at {:?}", binary_path))?;
-
-    #[cfg(not(windows))]
-    {
-        file.set_permissions(<fs::Permissions as fs::unix::PermissionsExt>::from_mode(
-            0o755,
-        ))
-        .await?;
-    }
-
-    let mut old_binary_paths = fs::read_dir(supermaven_dir()).await?;
-    while let Some(old_binary_path) = old_binary_paths.next().await {
-        let old_binary_path = old_binary_path?;
-        if old_binary_path.path() != binary_path {
-            fs::remove_file(old_binary_path.path()).await?;
-        }
-    }
-
-    Ok(binary_path)
+    std::env::var("SUPERMAVEN_AGENT")
+        .map(PathBuf::from)
+        .with_context(|| "Set environment variable SUPERMAVEN_AGENT with path to Supermaven Agent")
 }
