From e91cf37771d7c8dc193183492ab2a624e3d459d1 Mon Sep 17 00:00:00 2001
From: pancake <pancake@nopcode.org>
Date: Wed, 30 Apr 2025 19:21:12 +0200
Subject: [PATCH] Fix build with latest r2

---
 src/R2Scope.cpp | 11 ++++++++---
 1 file changed, 8 insertions(+), 3 deletions(-)

diff --git a/src/R2Scope.cpp b/src/R2Scope.cpp
index 128914f6..a10ade4d 100644
--- a/src/R2Scope.cpp
+++ b/src/R2Scope.cpp
@@ -151,10 +151,15 @@ FunctionSymbol *R2Scope::registerFunction(RAnalFunction *fcn) const {
 
 	auto symbollistElement = child(scopeElement, "symbollist");
 
-	ProtoModel *proto = fcn->cc ? arch->protoModelFromR2CC(fcn->cc) : nullptr;
+#if R2_VERSION_NUMBER >= 50909
+#define CALLCONV(x) (x)->callconv
+#else
+#define CALLCONV(x) (x)->cc
+#endif
+	ProtoModel *proto = CALLCONV(fcn) ? arch->protoModelFromR2CC(CALLCONV(fcn)) : nullptr;
 	if (!proto) {
-		if (fcn->cc) {
-			arch->addWarning ("Matching calling convention " + to_string(fcn->cc) + " of function " + to_string(fcn_name) + " failed, args may be inaccurate.");
+		if (CALLCONV(fcn)) {
+			arch->addWarning ("Matching calling convention " + to_string(CALLCONV(fcn)) + " of function " + to_string(fcn_name) + " failed, args may be inaccurate.");
 		} else {
 			arch->addWarning ("Function " + to_string(fcn_name) + " has no calling convention set, args may be inaccurate.");
 		}
