Author: Holger Jaekel <holger.jaekel@gmx.de>
Summary: use local clipp.h
----

--- a/tools/CMakeLists.txt
+++ b/tools/CMakeLists.txt
@@ -25,8 +25,6 @@
 # THE SOFTWARE.
 #

-find_package(Clipp CONFIG REQUIRED)
-
 add_executable(tiledb-cli EXCLUDE_FROM_ALL
   src/commands/help_command.cc
   src/commands/info_command.cc
@@ -45,7 +43,6 @@
   target_link_libraries(${TOOL} PRIVATE
     TILEDB_CORE_OBJECTS_ILIB
     TILEDB_CORE_OBJECTS
-    clipp::clipp
   )
   target_include_directories(${TOOL} PRIVATE
     ${CMAKE_CURRENT_SOURCE_DIR}/..


--- a/tools/src/commands/command.h
+++ b/tools/src/commands/command.h
@@ -33,7 +33,7 @@
 #ifndef TILEDB_CLI_COMMAND_H
 #define TILEDB_CLI_COMMAND_H
 
-#include <clipp.h>
+#include "clipp.h"
 
 namespace tiledb {
 namespace cli {


--- a/tools/src/main/tiledb.cc
+++ b0/tools/src/main/tiledb.cc
@@ -30,7 +30,7 @@
  * Command-line interface for common TileDB tasks.
  */
 
-#include <clipp.h>
+#include "commands/clipp.h"
 
 #include <iostream>
 #include <map>
