--- sentry-native-0.6.7/src/modulefinder/sentry_modulefinder_linux.c.orig
+++ sentry-native-0.6.7/src/modulefinder/sentry_modulefinder_linux.c
@@ -567,13 +567,9 @@
     }
 }
 
-// copied from:
-// https://github.com/google/breakpad/blob/216cea7bca53fa441a3ee0d0f5fd339a3a894224/src/client/linux/minidump_writer/linux_dumper.h#L61-L70
-#if defined(__i386) || defined(__ARM_EABI__)                                   \
-    || (defined(__mips__) && _MIPS_SIM == _ABIO32)
+#if UINTPTR_MAX == 0xFFFFFFFF
 typedef Elf32_auxv_t elf_aux_entry;
-#elif defined(__x86_64) || defined(__aarch64__) || defined(__powerpc64__)      \
-    || (defined(__mips__) && _MIPS_SIM != _ABIO32)
+#elif UINTPTR_MAX == 0xFFFFFFFFFFFFFFFFu
 typedef Elf64_auxv_t elf_aux_entry;
 #endif
 
